internal/pkg/template/project/file.go

```diff
@@ -0,0 +1,63 @@
+package project
```

Comment:
`.keboola/project.json` file is related to `local project` not `template` directly.

So please find a better path, for example `internal/pkg/project/cachefile`, 
and add some package comments, something like:
```
// Package cachefile manages .keboola/project.json file with cached infromation about project for local commands, without API access.
```